Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Builder
+FROM golang:alpine as builder
+
+WORKDIR /build
+COPY . /build
+
+# Vet and build
+RUN go vet
+RUN go build
+
+# Runner
+FROM golang:alpine
```

Comment:
We don't need the full Go compiler to run the app. In fact, I think the much smaller gcr.io/distroless/static will suffice.